drivers/gpu/drm/vc4/vc4_firmware_kms.c

```diff
@@ -272,6 +278,7 @@ struct vc4_fkms_encoder {
 	struct drm_encoder base;
 	bool hdmi_monitor;
 	bool rgb_range_selectable;
+	int display_num;
```

Comment:
I feel that this should be accessible from either the crtc or connector (the bits either side of the encoder).
In reality I get lost with the potential dereferencing between the structures, and this driver is fairly monolithic anyway.

Reply:
Fair enough. I know that finding the right linkings can be less than obvious.